src/io/flutter/utils/AnimatedIcon.java

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
This file is a pure copy of the matching file from IntelliJ which wasn't available for all versions we support. In the future we may want to tweak this class further to make the animations a bit more efficient.